gc-check-ins.md

```diff
@@ -0,0 +1,20 @@
+## Regular check-ins with SIG leads
+
+As defined in the [charter of the GC](./governance-charter.md#regular-check-ins-with-sig-leads) the GC will check in with maintainers of all long-term and short-term SIGs on a regular basis. During this check in,
+
+* any project files or roadmap entries should be reviewed to verify that they are still accurate.
+* if a SIG lead has any private concerns they wish to raise with the Governance Committee, this is an opportunity to do so.
+* the health of the SIG will be reviewed, and what measures can be applied to help the SIG.
+
+The check in process is as follows:
+
+* For each SIG a GC member is appointed as a GC liaison. The name of that person will be listed publicly.
+* If a GC member is the maintainer themselves, they are by default the GC liaison for this SIG.
+* The GC liaison is appointed for the duration of the project lifecycle, or until they do not get reelected into the GC. Outside of that regular duration, a GC liaison can resign by providing a named successor.
+* The GC liaison is responsible to check in with the maintainers at least once per month. This can either happen via a (private, non recorded) meeting or through a text-based conversation initiated by the GC liaison.
```

Comment:
We kind of touched the idea that WGs and SIGs are the same for the purposes of this proposal. In that case, what to do with WGs without named leaders, like the End-User WG?

Reply:
My suggestion would be that the SIG names those leaders then. What is not defined here I guess is how a SIG manages that relationship: are all maintainers in contact with the GC liaison or do they appoint someone as well, or are they doing something "rolling". I think this (a) does not need to be defined yet, and (b) might be different from SIG to SIG

Reply:
I will resolve this conversation, since it has no impact on this document right now.